parsl/executors/high_throughput/interchange.py

```diff
@@ -300,6 +301,21 @@ def _command_server(self, kill_event):
                                 'active': self._ready_manager_queue[manager]['active']}
                         reply.append(resp)
 
+                elif command_req == "BLOCKS":
```

Comment:
Some of this logic would be moved to `MANAGERS`, with `idle_duration` reporting 0 when manager is not idle, and a float otherwise, in seconds.